README.md

```diff
@@ -827,6 +827,10 @@ see the `-pass` command line option).
 The QEMU backend is run with the `-nographic` option by default. This
 may be changed with `export SPREAD_QEMU_GUI=1`.
 
+The bios that qemu uses can be changed via the SPREAD_QEMU_BIOS
+environment variable. To switch to an UEFI bios on Ubuntu
+`export SPREAD_QEMU_BIOS=/usr/share/OVMF/OVMF_CODE.ms.fd` can be used.
```

Comment:
This can't be used as is because the bios location is a property of the host. But the images used can be both efi/non-efi and some efi images may need to be booted with a different efi bios. So one global options like this is not granular enough.

Reply:
Have you considered adding a `bios:` string when necessary in `backends.systems` instead?

Reply:
Have you considered adding an optional `bios:` string for `backends.systems` in `spread.yaml` instead?